test/integration/query-interface/createTable.test.js

```diff
@@ -75,16 +74,18 @@ describe(Support.getTestDialectTeaser('QueryInterface'), () => {
           expect(indexes[1].unique).to.be.true;
           expect(indexes[1].fields[0].attribute).to.equal('name');
           break;
+
         case 'mariadb':
         case 'mysql':
-        // name + email
+        case 'db2':          
```

Comment:
@AWADHAMBIKA except this line, all other changes in this file should be removed.

Reply:
Done